parsec/core/backend_connection/cmds.py

```diff
@@ -594,6 +594,7 @@ async def pki_enrollment_submit(
     enrollment_id: UUID,
     force: bool,
     submitter_der_x509_certificate: bytes,
+    submitter_der_x509_certificate_email: str,
```

Comment:
`pki_enrollment_submit` is an anonymous command, so I think this function is never used (and same thing for `pki_enrollment_info`)

Reply:
That's right, i removed.This field has been moved to the anonymous command